packages/common-cosmos/src/project/utils.ts

```diff
@@ -50,3 +52,29 @@ export function isCosmosTemplates(
 ): templatesData is (RuntimeDatasourceTemplate | CustomDatasourceTemplate)[] {
   return (isRuntimeCosmosDs(templatesData[0]) || isCustomCosmosDs(templatesData[0])) && gte(specVersion, '0.2.1');
 }
+
+@ValidatorConstraint({name: 'isFileReference', async: false})
+export class FileReferenceImp implements ValidatorConstraintInterface {
+  validate(value: Map<string, FileReference>, args: ValidationArguments): boolean {
+    if (!value) {
+      return false;
+    }
+    Object.values(value).forEach((fileReference: FileReference) => {
+      if (!this.isValidFileReference(fileReference)) {
+        throw new Error(this.defaultMessage(args));
```

Comment:
This function seems to want `false` if its invalid but an error is being thrown here. Is that how the library expects you to handle invalid values?